test/models/users_test.rb

```diff
@@ -58,10 +59,58 @@ class UsersTest < ActiveSupport::TestCase
     end
 
     test "find_or_create_from_github accepts minimal users without name nor email" do
+      FakeWeb.register_uri(:get, "https://api.github.com/user/emails", status: %w(200 OK), body: [].to_json, content_type: "application/json")
       user = User.find_or_create_from_github(@minimal_github_user)
       assert_equal @minimal_github_user.login, user.login
     end
 
+    test "find_or_create_from_github selects any email when org email is unspecified" do
+      github_org_user = stub(
+        id: 42,
+        name: 'Jim Jones',
+        login: 'jim',
+        email: "jim@#{@org_domain}",
+        avatar_url: 'https://avatars.githubusercontent.com/u/42?v=3',
+        url: 'https://api.github.com/user/jim',
+      )
+
+      Shipit.preferred_org_emails = []
```

Comment:
I don't think it matters right now but to save us in future I think this should be set back to the previous value in an ensure block.